README.md

```diff
@@ -118,6 +120,31 @@ ggsave("plot.pdf", plot = p)
 embed_fonts("plot.pdf", outfile = "plot_embed.pdf")
 ```
 
+Add logos
+---------
+
+sagethemes allows you to add a logo to the bottom right of your plot
+with the `logo_image()` function. By default it will use the Sage
+Bionetworks logo, but you can also provide your own image, for example a
+project-specific logo. Currently due to limitations of the
+implementation, this must be the last function call in your ggplot2
+chain; you won’t be able to add additional ggplot2 elements after adding
+the logo.
```

Comment:
No, there's not really a way. patchwork takes over the `+` operator to allow you to combine multiple plots (or in this case plots and images), and it actually will let you use `+` to add items to the last plot. Unfortunately in our case the last "plot" is the image, so adding ggplot elements doesn't really have an effect. However there's no warning or error to catch.

Technically I suppose you could provide the image first, change the layout, and then keep adding more plot elements, but that seems more complicated than putting the logo at the end. Still I might reword the "must" to "should" since there is technically a workaround.

```r
logo_image() +
  ggplot(mpg, aes(displ, hwy, colour = class)) +
  geom_point() +
  logo_layout(design = "BBBBBB\n####AA") +
  theme_dark()
```

Reply:
Not worth the added complexity, IMHO. I inquired just in case it was a few lines that could be added. Thanks for taking the time to explain, @karawoo!